content/en/docs/next/quickstart.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Quickstart
+weight: 900
+description: Get etcd up and running in less than 5 minutes!
+---
+
+## Step 1 - Install etcd
+
+Set the environment variables to help with the commands that follow.
+
+```
+ETCD_VER={{< param git_version_tag >}}
+
+GOOGLE_URL=https://storage.googleapis.com/etcd
+GITHUB_URL=https://github.com/etcd-io/etcd/releases/download
+```
+
+Download and install etcd:
+
+On Linux
+```
+$ DOWNLOAD_URL=${GITHUB_URL}
+
+$ rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+$ rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+$ curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-linux-amd64.tar.gz -o /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
+                                 Dload  Upload   Total   Spent    Left  Speed
+100   644  100   644    0     0   1396      0 --:--:-- --:--:-- --:--:--  1396
+100 12.5M  100 12.5M    0     0  9767k      0  0:00:01  0:00:01 --:--:-- 17.7M
+
+$ tar xzvf /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz -C /tmp/test-etcd --strip-components=1
+etcd-v3.5.0-alpha.0-linux-amd64/etcdctl
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/rpc.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/v3lock.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/dev-guide/apispec/swagger/v3election.swagger.json
+etcd-v3.5.0-alpha.0-linux-amd64/Documentation/README.md
+etcd-v3.5.0-alpha.0-linux-amd64/READMEv2-etcdctl.md
+etcd-v3.5.0-alpha.0-linux-amd64/README.md
+etcd-v3.5.0-alpha.0-linux-amd64/README-etcdctl.md
+etcd-v3.5.0-alpha.0-linux-amd64/etcd
+```
+
+On macOS
+```
+$ DOWNLOAD_URL=${GOOGLE_URL}
+
+$ rm -f /tmp/etcd-${ETCD_VER}-linux-amd64.tar.gz
+$ rm -f /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+$ rm -rf /tmp/test-etcd && mkdir -p /tmp/test-etcd
+
+$ curl -L ${DOWNLOAD_URL}/${ETCD_VER}/etcd-${ETCD_VER}-darwin-amd64.zip -o /tmp/etcd-${ETCD_VER}-darwin-amd64.zip
+$ unzip /tmp/etcd-${ETCD_VER}-darwin-amd64.zip -d /tmp
+$ mv /tmp/etcd-${ETCD_VER}-darwin-amd64/* /tmp/test-etcd
+```
+
+## Step 2 - Run etcd
+
+In order to test etcd, two terminals are needed. One to run the etcd service, another to run etcdctl.
+
+In one terminal, go to the download directory and start etcd:
+```
+$ cd /tmp/test-etcd
+$ etcd --logger=zap
```

Comment:
Using the `--logger=zap` flag here as just calling `etcd` generates a deprecation warning.

Reply:
I'm not getting any deprecation warning for 3.5.

Reply:
This may have be an artifact from when I was using v3.4. I'll recheck and update if it's not needed.

Reply:
Here's what I'm getting on Ubuntu:
```
➜  test-etcd etcd
[WARNING] Deprecated '--logger=capnslog' flag is set; use '--logger=zap' flag instead
2021-05-13 17:15:47.396640 I | etcdmain: etcd Version: 3.4.15
2021-05-13 17:15:47.396695 I | etcdmain: Git SHA: aa7126864
2021-05-13 17:15:47.396702 I | etcdmain: Go Version: go1.12.17
2021-05-13 17:15:47.396709 I | etcdmain: Go OS/Arch: linux/amd64
2021-05-13 17:15:47.396716 I | etcdmain: setting maximum number of CPUs to 4, total number of available CPUs is 4
2021-05-13 17:15:47.396725 W | etcdmain: no data-dir provided, using default data-dir ./default.etcd
[WARNING] Deprecated '--logger=capnslog' flag is set; use '--logger=zap' flag instead
2021-05-13 17:15:47.398346 I | embed: name = default
2021-05-13 17:15:47.398392 I | embed: data dir = default.etcd
2021-05-13 17:15:47.398403 I | embed: member dir = default.etcd/member
2021-05-13 17:15:47.398410 I | embed: heartbeat = 100ms
2021-05-13 17:15:47.398417 I | embed: election = 1000ms
2021-05-13 17:15:47.398423 I | embed: snapshot count = 100000
2021-05-13 17:15:47.398435 I | embed: advertise client URLs = http://localhost:2379
2021-05-13 17:15:47.404630 I | etcdserver: starting member 8e9e05c52164694d in cluster cdf818194e3a8c32
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d switched to configuration voters=()
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d became follower at term 0
raft2021/05/13 17:15:47 INFO: newRaft 8e9e05c52164694d [peers: [], term: 0, commit: 0, applied: 0, lastindex: 0, lastterm: 0]
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d became follower at term 1
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d switched to configuration voters=(10276657743932975437)
2021-05-13 17:15:47.408294 W | auth: simple token is not cryptographically signed
2021-05-13 17:15:47.410123 I | etcdserver: starting server... [version: 3.4.15, cluster version: to_be_decided]
2021-05-13 17:15:47.412215 I | etcdserver: 8e9e05c52164694d as single-node; fast-forwarding 9 ticks (election ticks 10)
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d switched to configuration voters=(10276657743932975437)
2021-05-13 17:15:47.413078 I | etcdserver/membership: added member 8e9e05c52164694d [http://localhost:2380] to cluster cdf818194e3a8c32
2021-05-13 17:15:47.413826 I | embed: listening for peers on 127.0.0.1:2380
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d is starting a new election at term 1
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d became candidate at term 2
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d received MsgVoteResp from 8e9e05c52164694d at term 2
raft2021/05/13 17:15:47 INFO: 8e9e05c52164694d became leader at term 2
raft2021/05/13 17:15:47 INFO: raft.node: 8e9e05c52164694d elected leader 8e9e05c52164694d at term 2
2021-05-13 17:15:47.508601 I | etcdserver: setting up the initial cluster version to 3.4
2021-05-13 17:15:47.509770 I | etcdserver: published {Name:default ClientURLs:[http://localhost:2379]} to cluster cdf818194e3a8c32
2021-05-13 17:15:47.509826 I | embed: ready to serve client requests
2021-05-13 17:15:47.511176 N | etcdserver/membership: set the initial cluster version to 3.4
2021-05-13 17:15:47.511273 I | etcdserver/api: enabled capabilities for version 3.4
2021-05-13 17:15:47.511915 N | embed: serving insecure client requests on 127.0.0.1:2379, this is strongly discouraged!
```

Will check macOS shortly.

Reply:
Interesting -- when testing on macOS, I definitely needed the `./`, and there's no deprecation warning. When I added the `./` to test on Ubuntu (adding `./` to make things consistent across platforms would make it cleaner), the deprecation warning went away.